README.md

```diff
@@ -3,3 +3,32 @@
 The goal of this wiki is to modernize the information found in the long-standing OSDev wiki found at [wiki.osdev.org](https://wiki.osdev.org) and to decentralize it. It is currently **WIP** and **not ready** for use.
 
 This wiki is not endorsed by, does not have any connection to, and is not related in any way with another wiki found at [wiki.osdev.org](https://wiki.osdev.org).
+
+# Building
+
+Thanks for contributing! 
+
+## Deps
+
+This wiki is written in Haskell. You will need a [Stack](https://docs.haskellstack.org/en/stable/README/) installation to compile and run the main binary.
+
+The project also depends on some non-Haskell deps. To install these, you need a [Python](https://www.python.org/downloads/), [Ruby](https://www.ruby-lang.org/fr/documentation/installation/) and [Rust](https://www.rust-lang.org/tools/install) installation:
```

Comment:
... need a Python ... => ... need Python ...

less sure about this one: ... need ... => ... will need ...